cli/azd/cmd/testdata/TestUsage-azd-env-get-value.snap

```diff
@@ -0,0 +1,19 @@
+
+Get specific environment value.
+
+Usage
+  azd env get-value <keyName> [flags]
+
+Flags
+        --docs               	: Opens the documentation for azd env get-value in your web browser.
+    -e, --environment string 	: The name of the environment to use.
+    -h, --help               	: Gets help for get-value.
+
+Global Flags
+    -C, --cwd string 	: Sets the current working directory.
+        --debug      	: Enables debugging and diagnostics logging.
+        --no-prompt  	: Accepts the default value instead of prompting, or it fails if there is no default.
```

Comment:
I think you always list this, but I assume its not relevant for this one (no prompting, right?)

Reply:
Yes always listed